debug: false
user_input:
  goals: '[("on", True, ("cupboard", "cube1"))]'
  demo_sequence: '[]'
  demo_parameters: '[]'
predicates:
  at:
    max_distance: 1.0
  on-pred:
    max_above: 0.03
explorer:
  search_budget_sec: 900
  time_proportion_demo: 0.8
  time_proportion_prepend: 0.8
  time_proportion_generalize: 0.8
  alternating_sequence_length: false
  min_sequence_length: 4
  max_sequence_length: 4
  max_failed_samples: 50
  bounding_box_inflation_length: 0.1
  action_denylist: []
  predicate_precondition_allowlist: ["inside", "on"]
  radii: [0.1, 1.0]
robot_arm:
  max_force_magnitude: 150
grasping:
  pregrasp_z_offset: 0.15
mcts:
  search_budget_sec: 900
  max_depth: 16
  closeby_objects_distance_threshold: 0.1
  avoid_double_nav: true
